app/routes/auth/github/callback.tsx

```diff
@@ -0,0 +1,9 @@
+import { LoaderFunction } from "@remix-run/node";
+import { authenticator } from "~/services/auth.server";
+
+export const loader: LoaderFunction = ({ request }) => {
+  return authenticator.authenticate("github", request, {
+    successRedirect: "/dashboard",
```

Comment:
we got a config.ts in the root with paths

Reply:
I fixed this :)